kuma/javascript/src/app.jsx

```diff
@@ -14,12 +15,14 @@ type Props = { documentData: DocumentData, requestData: RequestData };
 
 export default function App(props: Props) {
     return (
-        <LocaleProvider locale={props.requestData.locale}>
-            <DocumentProvider initialDocumentData={props.documentData}>
-                <UserProvider>
-                    <Page />
-                </UserProvider>
-            </DocumentProvider>
-        </LocaleProvider>
+        <GAProvider>
```

Comment:
A new component that provides the ga() function to any components that want to send data to google analytics. If google analytics is not configured, the provider will just provide a dummy function that doesn't do anything.